### PR TITLE
front: show user apps and devices

### PR DESCRIPTION
**Pull request by sh19910711**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@
     '/signin',
     '/signup',
     '/signout',
+    '/apps',
```
Comment (seiyanuta) [inline config/routes.rb]: Can you isolate frontend from the Rails app?

File: frontend/app/components/navbar.vue [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
-      .nav-left
-        h1.nav-brand
-          router-link(to='/') codestand.io
-      .nav-right
-        ul.nav-list
-          li
-            router-link(to='/signin') Sign in
-          li
-            router-link(to='/signup') Sign up
+<template>
```
Comment (seiyanuta) [inline frontend/app/components/navbar.vue]: Can you divide `<foo>.vue` into `<foo>.js`, `<foo>.html`, and `<foo>.css`? I want to write  *pure* ones.

Comment (sh19910711) [inline frontend/app/components/navbar.vue]: OK, let's try it. 👍

File: config/routes.rb (hunk 1 above)
Comment (sh19910711) [inline config/routes.rb]: Yup, I will create a new repository later.

**Comment by seiyanuta** [comment]
Multiple repositories is annoying. Just use webpack dev server or something.

**Comment by sh19910711** [comment]
> Multiple repositories is annoying. Just use webpack dev server or something.

Ah, I got it. Let's put something like that.

**Comment by sh19910711** [comment]
Isolated.

**Comment by seiyanuta** [comment]
awesome 👍

**Pull request merged**